api/src/main/java/me/lokka30/treasury/api/economy/response/EconomyFailureReason.java

```diff
@@ -40,10 +48,11 @@
          */
         @Override
         public @NotNull String getDescription() {
-            return "That feature is currently not available during migration!";
+            return "That feature is currently not available during migration.";
```

Comment:
Consistency nitpick, if L33 is changing to "The feature" then L51 should also be "The feature"